src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionServiceTestBase.java

```diff
@@ -944,18 +945,73 @@ public void givenWelsh_GeneratedDateIsAlreadySet_thenDoNotSetNewGeneratedDate()
         assertEquals(LocalDate.now(), payload.getGeneratedDate());
     }
 
+    @Test
+    public void givenPostHearingsIsTrue_thenShowIssuedDateOnDocument() {
+        setDescriptorFlowIndicator(isDescriptorFlowSupported() ? "yes" : "no", sscsCaseData);
+        sscsCaseData.getSscsFinalDecisionCaseData().setWriteFinalDecisionGenerateNotice("yes");
+        setHigherRateScenarioFields(sscsCaseData);
+        sscsCaseData.getSscsFinalDecisionCaseData().setWriteFinalDecisionDateOfDecision("2018-10-10");
+        sscsCaseData.getPostHearing().getCorrection().setCorrectionFinalDecisionInProgress(YesNo.YES);
+
+        service.preview(callback, DocumentType.CORRECTION_GRANTED, USER_AUTHORISATION, true, true, true);
+
+        NoticeIssuedTemplateBody payload = verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appellant Lastname", null, "2018-10-10", true,
```

Comment:
Save this date and name as String constants - they are used throughout the test class

Reply:
done